### PR TITLE
Rel v4.17.0.3 add index

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: Przemyslaw Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -420,7 +489,7 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.17
               <li>Known issues:</li>
               <ol>
                   <li><a href="https://github.com/pcengines/apu2-documentation/issues/308">APU7 iPXE network boot with i225 NICs does not work</a></li>
-                  <li><a href="https://github.com/pcengines/coreboot/issues/329">apuled driver doesn't work in FreeBSD.</a> Check the <a target="_blank" href="https://github.com/pcengines/apu2-documentation/blob/master/docs/gpios.md">GPIOs</a> document for workaround.</li>
+                  <li><a href="https://github.com/pcengines/coreboot/issues/329">apuled driver doesn't work in FreeBSD.</a> Check the <a target="_blank" href="https://github.com/pcengines/apu2-documentation/blob/master/docs/gpios.md"> GPIOs</a> document for workaround.</li>
```
Comment (miczyg1) [inline index.html:492]: Please revert this unnecessary change

File: index.html [hunk 2]
```diff
@@ -494,7 +563,7 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.16
...
```
Comment (miczyg1) [inline index.html:566]: Please revert this unnecessary change

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (sulewskiprzemyslaw) [reply index.html]: @miczyg1 this point have been changed, I add `&nbsp;` and space before `GPIOs` and `mPCIe`. Only with such a declaration the space in between two word is visible (I rely on locally built documentation).

File: index.html (hunk 2 above)
Comment (sulewskiprzemyslaw) [reply index.html:566]: @miczyg1 this point have been changed, I add `&nbsp;` and space before `GPIOs` and `mPCIe`. Only with such a declaration the space in between two word is visible (I rely on locally built documentation).

**Review by miczyg1 — APPROVED** [review]
